docs/modules/install/pages/index.adoc

```diff
@@ -110,6 +110,39 @@ You can configure it with `crontab -e`, for instance if you've created your Deci
 
 # Generate reminders
 4 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:reminders:all
+# Send notification mail digest daily
+5 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_daily
+
+# Send notification mail digest weekly on saturdays
+5 0 * * 6 cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_weekly
+----
+
+==== Notifications digest
+This is an email with all the notifications activity summarized. It's generated for the users that
+have checked the `email_on_notification` option in their user panel. Also they can set the frequency they want
+to receive the emails in the `notifications_sending_frequency` option.
+
+There are two tasks to send the mail notifications digest. The email will be send depending on when you set the job in the crontab.
+
+
+This one sends a daily summary for the users. It takes the current day when the task is triggered
+----
+decidim:mailers:notifications_digest_daily
+----
+
+And this one, a weekly sumary. It takes the range between the time the task has been triggered and 7 days ago. It should be run in a weekly basis in cron (this is by setting the last prameter of the
+cron format, which referes to the day of week).
+----
+decidim:mailers:notifications_digest_weekly
+----
+
+[source,console]
+----
+# Send notification mail digest daily
+5 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_daily
+
+# Send notification mail digest weekly on saturdays
+5 0 * * 6 cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_weekly
```

Comment:
Please remove this. As mentioned in the [other comment](https://github.com/decidim/decidim/pull/8833/files#r864879705), this page must be kept short.

Reply:
Done!